### PR TITLE
Storybook 7

### PR DESCRIPTION
**Pull request by ahuth**
Storybook 7 support!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+Storybook 7 is now officially supported!
+
+- [breaking] Drop Storybook 6 support
```
Comment (ahuth) [inline CHANGELOG.md:7]: Technically these changes will still actually work with Storybook v6. But I'm not going to test on that version, and figured I'll just say we only support Storybook 7.

File: src/browser/StorybookPage.ts [hunk 2]
```diff
@@ -96,13 +99,16 @@ function fetchStoriesFromWindow(): Promise<StorybookStory[]> {
 function emitSetCurrentStory(id: string) {
   const storybookPreview = window.__STORYBOOK_PREVIEW__;
 
+  // @ts-expect-error Access the protected "channel", so we can send stuff through it.
+  const channel = storybookPreview.channel;
```
Comment (ahuth) [inline src/browser/StorybookPage.ts:103]: `channel` is now "protected" (can only be accessed by subclasses, not externally). Hacking around that here, so we can use it.

**Review by jeremiah-clothier — APPROVED** [review]

File: demo/.babelrc.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      {
+        "targets": {
+          "chrome": 100
+        }
+      }
+    ],
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```
Comment (jinlee93) [inline demo/.babelrc.json:15]: Sorry, can you explain why this babel config is required?

Comment (ahuth) [reply demo/.babelrc.json:15]: Storybook 6 provided some babel config. v7 no longer does, and you have to provide your own to render the React components and whatnot.

**Review by jinlee93 — APPROVED** [review]

**Pull request merged**